src/main/java/people/employees/Developer.java

```diff
@@ -111,9 +107,9 @@ private LinkedList<TechStack> generateSkills() {
 
     private int getBaseSalary() {
         return switch (seniority) {
-            case SENIOR -> BASE_SENIOR_SALARY;
-            case MID -> BASE_MID_SALARY;
-            case JUNIOR -> BASE_JUNIOR_SALARY;
+            case SENIOR -> 2137;
+            case MID -> 2137;
+            case JUNIOR -> 2137;
```

Comment:
papaj is watching you
![image](https://user-images.githubusercontent.com/2371681/198058931-deb8c0d1-f310-4b73-a371-2f883a6e0f8d.png)

Reply:
When you looking at the papaj, papaj is looking at you 
~Karollo Wojtyllo